tsconfig.test.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "outDir": "build/test",
+    "rootDir": "test"
+  },
+  "include": ["test/*", "test/**/*"]
```

Comment:
Yeah probably good call to separate these

Reply:
I recognize that, in other worlds (e.g., Go, Ruby), placing tests next to source files is common; in Node.js land, it isn't for reasons of separation of concerns. Plus, UI tests cross file divides and don't fit nicely into the `foo.spec.ts` paradigm where `foo` refers to `foo.ts`, a source file.